x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/AsyncOperator.java

```diff
@@ -34,6 +35,11 @@
  * to reduce communication overhead and fetches a {@code Fetched} at a time.
  * It's the responsibility of subclasses to transform that {@code Fetched} into
  * output.
+ * <p>
+ *     This operator will also take care of merging response headers from the thread context into the main thread,
+ *     which <b>must</b> be the that closes this.
```

Comment:
I think there's something wrong with the end of this sentence. It doesn't make sense to me.

Reply:
"which must</b> be the **_one_** that closes this". Fixed!